### DISCUSSION TITLE
v3.7.1-R4.2

### DISCUSSION THREAD

**Discussion by RyanMetcalfeInt8** [intel/openvino-plugins-ai-audacity#346] [Announcements]
# Download :floppy_disk:  
Download the Windows installer: [audacity-win-v3.7.1-R4.2-64bit-OpenVINO-AI-Plugins.exe](https://github.com/intel/openvino-plugins-ai-audacity/releases/download/v3.7.1-R4.2/audacity-win-v3.7.1-R4.2-64bit-OpenVINO-AI-Plugins.exe)

# New Feature! Audio Super Resolution :sparkles: 
* Upscales and enriches audio for improved clarity and detail. 
    * Documentation available [here](https://github.com/intel/openvino-plugins-ai-audacity/blob/87d5d4dea03a53b9a91893e20e7c10224e50109f/doc/feature_doc/super_resolution/README.md) 
    * It is a port of this amazing project: https://github.com/haoheliu/versatile_audio_super_resolution 
    
## Other Changes since previous release:
* Updated OpenVINO™ to 2024.6.0

# Compatibility :wrench:
:warning: This plugin release is *only* compatible with [Audacity 3.7.1](https://github.com/audacity/audacity/releases/tag/Audacity-3.7.1) 64-bit Release for Windows -- so make sure you have this version of Audacity installed! 

## :warning: If you've previously installed any of our 3.7.X, 3.6.X, or 3.5.X plugin releases, the recommendation for upgrade is to *not* uninstall the previous version. Instead, just download this version and models that you have not installed yet (such as Super Resolution). This way, you won't need to re-download all of the models again.

:warning: **Make sure to select an Audacity 3.7.1 location during installation. If you have multiple Audacity versions installed, the installer may select the wrong location by default -- so be mindful of this, and manually adjust this using the *browse* button if needed**. 

:penguin: Although we are only providing an installer for Windows right now, it can be compiled for Linux using [this build procedure](https://github.com/intel/openvino-plugins-ai-audacity/blob/87d5d4dea03a53b9a91893e20e7c10224e50109f/doc/build_doc/linux/README.md).

# Supported Hardware
As this release is built on top of OpenVINO™ 2024.6.0, it will generally support hardware defined in this list: https://docs.openvino.ai/2024/about-openvino/release-notes-openvino/system-requirements.html 

# Need help? :raising_hand:
If you encounter any issues during install procedure, reach out to us for help by filing an issue [here](https://github.com/intel/openvino-plugins-ai-audacity/issues). We are happy to help!

<hr /><em>This discussion was created from the release <a href='https://github.com/intel/openvino-plugins-ai-audacity/releases/tag/v3.7.1-R4.2'>v3.7.1-R4.2</a>.</em>

**Comment by Samethe168** [comment]
Unfortunately, the program crashed and closed halfway through. I had selected multiple modules, and it is taking up nearly 10GB of space on my disk. Since the installation was not completed, it is unclear where the files are located, and I cannot delete them. Could you please help me?"

**Reply by RyanMetcalfeInt8** [reply]
Hmm, 'the program' here is the installer? 

The installer downloads files to `C:\Users\<Username>\AppData\Local\Temp\is-XXXXX.tmp\`, where `XXXXX` would be some random combination of letters and numbers -- so perhaps you can look for that folder.

Depending on how far the installer got before it crashed, it may have started to unpack some of the zip files to `C:\Program Files\Audacity\openvino-models` -- so you can also check there.

**Comment by lerdixxx** [comment]
Hi there, why there no updates anymore missing Music Seperation (8 stem) kick,Hi-Hat, Snare, Bass, Vocal, Instrument, Percs, Crashes with (4 Stem) Drums, Vocal, Bass, Instrument make me difficult to work

**Reply by RyanMetcalfeInt8** [reply]
Which 8-stem Music Separation model are you referring to?  An open source one exists?

**Reply by lerdixxx** [reply]
hi thanks for the quick reply , there is STEMwerk 2.3.0.6 and 2.3.0.7 using in DAW Reaper , but cant use at all what make me annoyed a lot .I prefer audaCity and OpenVino much more easiar to work with it.
<img width="367" height="255" alt="Screenshot 2026-08-06 063616" src="https://github.com/user-attachments/assets/cae53d3e-25c3-4d9c-ad89-021c954b7a70" />

**Reply by RyanMetcalfeInt8** [reply]
Hmm, I'm not familiar with the models that STEMwerk uses. Our next release will have many new stem separation models:
<img width="896" height="495" alt="image" src="https://github.com/user-attachments/assets/ca7486a2-031f-436f-8136-457361abf45c" />
